src/drawbridge.rs

```diff
@@ -179,6 +180,16 @@ pub fn client(
     Ok(cl)
 }
 
+fn http_client(mut req: oauth2::HttpRequest) -> Result<oauth2::HttpResponse, oauth2::ureq::Error> {
+    req.headers.insert(
+        "User-Agent",
+        format!("enarx-cli/{}", env!("CARGO_PKG_VERSION"))
+            .try_into()
+            .unwrap(),
+    );
+    ureq_http_client(req)
```

Comment:
Why not just `ureq::http_client` or `oauth2::ureq::http_client` to avoid having to rename the import?

Reply:
Don't know, that was just personal feeling. I fixed it now, thanks!